treeherder/log_parser/management/commands/crossreference_missing.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import time
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
For some reason `ProcessPoolExecutor` had problems; in the subprocesses the database connecgtion didn't seem to be correctly initialized.